### ISSUE TITLE
Update SvelteKit realword with breaking change "Removing session #5883" and persistent login

### ISSUE THREAD

**Issue by 3daddict** [sveltejs/realworld#135]
I am having a hard time getting persistent login to work with the SvelteKit Removing session [#5883](https://github.com/sveltejs/kit/discussions/5883) update. On page reload my app does not set a cookie and redirects back to the login page. Seems like the `src/routes/auth/login/+server.js` is not setting the cookie. Are there any plans to update this repo to the latest SvelteKit breaking changes updates?

Are there any existing examples I can reference?
I am looking for guidance on how the `src/routes/auth/login/+server.js` is called and how the cookie is persistently read through out page routes.

*src/routes/auth/_respond.js*
```js
import { error, json } from '@sveltejs/kit';
import { Buffer } from 'buffer';

export function respond(body) {
	if (body.errors) {
		throw error(401, body);
	}

	const value = Buffer.from(JSON.stringify(body)).toString('base64');

	return json(body, {
		headers: {
			'set-cookie': `jwt=${value}; Path=/; HttpOnly`
		}
	});
}
```

*src/routes/auth/login/+server.js*
```js
import * as api from '$utils/api';
import { respond } from '../_respond';

export async function POST({ request }) {
  const json = await request.json();
  const body = await api.post('auth/login', {
    email: json.email,
    password: json.password,
  });

  return respond(body);
}

```

*src/routes/login/+page.js*
```js
import { redirect } from '@sveltejs/kit';
import { page } from '$app/stores';

export async function load({ page }) {
	if (page.user) {
		throw redirect(302, '/');
	}

	return {};
}
```

*src/routes/login/+page.svelte*
```html
<script>
	import { page } from '$app/stores';
	import { goto } from '$app/navigation';
	import { post } from '$lib/utils.js';

	let email = '';
	let password = '';
	let errors = null;

	async function submit(event) {
		const response = await post(`auth/login`, { email, password });

		if (response.user) {
			page.user = response.user;
			goto('/');
		}
	}
</script>
```

**Comment by 3daddict** [comment]
I found this had to do with the migration of session to page and the new method of loading. I am working on a PR here: https://github.com/sveltejs/realworld/pull/136 and have some of it working so far.

**Comment by Rich-Harris** [comment]
fixed in #139
